competency-change.md

```diff
@@ -0,0 +1,13 @@
+Broad InnerSource adoption an example of a competency change.
+Some competency (in this example InnerSource) should be upleveled across the company.
+
+Here are the roles involved in running a competency change effort.
+
+| **Role** | **Description** |
+|----------|-----------------|
+| **CTO** | This person enables the competency program driver to represent the CTO in driving improvement in the competency across the company. |
+| **Driver** | This person is enabled by the CTO to drive improvement in the competency across the company. |
+| **VP** | Works with the CTO directly.  These people know and believe that the CTO is enabling the competency program driver. |
+| **Champion** | Each VP has at least one champion.  The VP enables this person to drive improvement in the competency across their area. |
+| **Team Leader** | People in leadership/influence position on an individual team. Could be engineering/solution/scrum manager, etc. |
+| **Team Member** | This person is on a regular team at the company.  We want these regular teams to improve in the given competency. |
```

Comment:
In addition to the textual description, a graph might help to understand both roles and their relationships?

```mermaid
graph TD;
    A((CTO)) -- enables --> B((Driver))
    B((Driver)) -- drives competency --> Org[Organization]
    A((CTO)) --> C((VP))
    C((VP)) -- enables --> D((Champion))
    D((Champion)) -- drives competency --> Area[Area]

    E((Team Lead)) --> F((Team Member))
```

Creating the draft of that graph also raised some questions:

- relationship between Driver and Champion
- relationship between Champion and Team Lead

Reply:
Thank you for this suggestion!  I am glad that you have this interest in creating pictures!  It is very helpful when trying to communicate a concept.  I added a picture - see what you think.

Reply:
I like it! 

The "holds accountable" vs "accountable to" might be a bit confusing
e.g. Champion holds the Team Leader accountable. However the Team Leader is accountable to the VP. I guess with the latter you mean the reporting hierarchy, while the former is purely related to the InnerSource Program?

Reply:
That is correct.  Updated the diagram.